packs/data/spells.db/umbral-mindtheft.json

```diff
@@ -0,0 +1,99 @@
+{
+    "_id": "uopaLE01meX11Mbw",
+    "data": {
+        "ability": {
+            "value": ""
+        },
+        "area": {
+            "areaType": "",
+            "value": ""
+        },
+        "category": {
+            "value": "spell"
+        },
+        "components": {
+            "focus": false,
+            "material": false,
+            "somatic": true,
+            "verbal": true
+        },
+        "cost": {
+            "value": ""
+        },
+        "damage": {
+            "value": {
+                "3fs8oc0vhfp42yxz": {
+                    "applyMod": false,
+                    "type": {
+                        "categories": [],
+                        "subtype": "",
+                        "value": "mental"
+                    },
+                    "value": "2d6"
+                }
+            }
+        },
+        "description": {
+            "value": "<p>You prepare to steal a broad field of knowledge from another creature, siphoning it from their mind and storing it in a pocket of the Shadow Plane connected to your own mind. During the spell’s duration, you can use the @Compendium[pf2e.actionspf2e.Steal]{Steal} action to attempt to take one Lore skill proficiency from a creature instead of Stealing an item. You can also make one attempt to Steal as part of Casting the Spell. If you succeed at your check to Steal a Lore skill, you deal 2d6 mental damage to the target if they were trained in that skill, plus [[/r {2d6}[mental ]]]{2d6 additional mental damage} per proficiency rank they have in that skill above trained, as the sudden loss of knowledge leaves an aching absence in the target’s mind. Determine the Lore skill at random from among those known by the target; on a critical success, you learn what Lore skills the creature is proficient in and can select the skill rather than choosing at random. If the target isn’t trained in any Lore skills, you get nothing, but you can continue to attempt to Steal Lore from other creatures for the remaining duration. The target loses the benefits of its training in that Lore skill, and you gain their proficiency rank in that Lore skill. This effect lasts until your next daily preparations or until you are incapacitated, in which case the Lore siphons back into the target creature’s mind from the Shadow Plane, wherever you both may be. As normal, you can Steal only from a target within your reach. Once you’ve stolen one Lore skill, you can’t attempt to Steal more Lore. If you succeed at your Thievery check, you must take the Lore you stole.</p>"
```

Comment:
```suggestion
            "value": "<p>You prepare to steal a broad field of knowledge from another creature, siphoning it from their mind and storing it in a pocket of the Shadow Plane connected to your own mind. During the spell's duration, you can use the @Compendium[pf2e.actionspf2e.Steal]{Steal} action to attempt to take one Lore skill proficiency from a creature instead of Stealing an item. You can also make one attempt to Steal as part of Casting the Spell. If you succeed at your check to Steal a Lore skill, you deal 2d6 mental damage to the target if they were trained in that skill, plus [[/r {2d6}[mental ]]]{2d6 additional mental damage} per proficiency rank they have in that skill above trained, as the sudden loss of knowledge leaves an aching absence in the target's mind. Determine the Lore skill at random from among those known by the target; on a critical success, you learn what Lore skills the creature is proficient in and can select the skill rather than choosing at random. If the target isn't trained in any Lore skills, you get nothing, but you can continue to attempt to Steal Lore from other creatures for the remaining duration. The target loses the benefits of its training in that Lore skill, and you gain their proficiency rank in that Lore skill. This effect lasts until your next daily preparations or until you are incapacitated, in which case the Lore siphons back into the target creature's mind from the Shadow Plane, wherever you both may be. As normal, you can Steal only from a target within your reach. Once you've stolen one Lore skill, you can't attempt to Steal more Lore. If you succeed at your Thievery check, you must take the Lore you stole.</p>"
```